dags/data_retention_reallocate_dag.py

```diff
@@ -58,18 +59,10 @@ def generate_sql_reallocate(policy):
 def data_retention_reallocate():
     policies = map_policy.expand(policy=get_policies())
 
-    reallocate = PostgresOperator.partial(
+    PostgresOperator.partial(
```

Comment:
I notice this DAG is not included in https://github.com/crate/crate-airflow-tutorial/pull/74/files
Should we move to SQLExecuteQueryOperator here as well?

Reply:
Strang that `pytest` didn't show any deprecation warnings for that DAG. But I adjusted it in the Airflow 2.5 branch as well and rebased this one.